pkg/functions/docker/driver.go

```diff
@@ -248,7 +248,7 @@ func (d *Driver) findActiveContainer(ctx context.Context, functionID, functionRe
 // GetRunnable creates runnable representation of the function
 func (d *Driver) GetRunnable(e *functions.FunctionExecution) functions.Runnable {
 	return func(ctx functions.Context, in interface{}) (interface{}, error) {
-		bytesIn, _ := json.Marshal(functions.Message{Context: ctx, Payload: in})
+		bytesIn, _ := json.Marshal(in)
```

Comment:
I don't think it's this simple.  How are secrets passed?

Reply:
Then how about modify funky, to remove non-necessary field before call python image?
@tenczar

Reply:
For Knative Funky was modified to read the secrets from environment variables. This means that a function gets secrets passed to it at creation time. When a function is invoked, funky reads the secrets from the environment variables and adds them to the context that is passed to the language server.